packages/filesystem/src/node/nsfw-watcher/nsfw-filesystem-watcher.ts

```diff
@@ -79,7 +79,7 @@ export class NsfwFileSystemWatcherServer implements FileSystemWatcherServer {
         const basePath = FileUri.fsPath(uri);
         this.debug('Starting watching:', basePath);
         if (fs.existsSync(basePath)) {
-            await this.start(watcherId, basePath, options);
```

Comment:
we should implement the cancellation then, otherwise, it can be that `unwatchFileChanges` is called, do nothing and then watching actually started (which can be stopped anymore)